src/NDItk/base/IntegerListRecord/test/IntegerListRecord.test.cpp

```diff
@@ -23,7 +24,7 @@ SCENARIO( "IntegerListRecord" ) {
 
     WHEN( "the data is given explicitly" ) {
 
-      IntegerListRecord chunk( "rprod_all", { 1, 2, 3, 4, 5, 6, 7, 8 } );
+      IntegerListRecord chunk( Keyword( "rprod_all" ), { 1, 2, 3, 4, 5, 6, 7, 8 } );
```

Comment:
You have rprod_all here. But this is just testing the types of integerlists, right?

Reply:
Yes, this was just for testing. I just copied something out of some of the pdfs you sent me. I didn't even remember this was the keyword I used.